ci/Dockerfile

```diff
@@ -1,59 +1,49 @@
-# This file is adapted from Spicy upstream's `docker/Dockerfile.ubuntu-20`.
-# Fixes here should probably also happen in that file.
+FROM debian:bullseye-slim AS build
 
-FROM ubuntu:focal
-
-# Increase this to force a rebuild of the Docker image (in Cirrus, in particular).
-ENV IMAGE_VERSION=1
```

Comment:
Absicht, dass das raus ist?

Reply:
I removed this to stay relatively close to the `Dockerfile` used in zeek/zeek. I believe we added this as a workaround for incorrect image caching in Cirrus (which I suspect might be fixed since I haven't seen it for some time), but we could always add a similar line if we want to force a rebuild (this would need a new commit in any case).